modules/nf-core/modules/preseq/lcextrap/main.nf

```diff
@@ -1,6 +1,6 @@
 process PRESEQ_LCEXTRAP {
```

Comment:
any idea why the `meta.yml` file for this module is empty?
it is not empty in modules: https://github.com/nf-core/modules/blob/master/modules/preseq/lcextrap/meta.yml

Reply:
Its' not empty?
https://github.com/nf-core/rnaseq/blob/dev/modules/nf-core/modules/preseq/lcextrap/meta.yml